### ISSUE TITLE
I wish make ``rotatescreen`` implementation for macOS.

### ISSUE THREAD

**Issue by Alexandro1112** [danny-burrows/rotate-screen#4]
Can I become project creator, from adding Mac-os version of screen rotation?

**Comment by danny-burrows** [comment]
Of course! That would be excellent! If you'd like to create a fork and submit your pull request I'd be happy to take a look. :)

**Comment by Alexandro1112** [comment]
So, I get to work today!

**Comment by Alexandro1112** [comment]
Good evening. I already wrote Mac version. I need create a fork?

**Comment by danny-burrows** [comment]
Yes, if you'd like to submit a pull request you can create a fork and I'll take a look

**Comment by Alexandro1112** [comment]
I created fork. Please unlock repository
